src/shared/conf-parser.c

```diff
@@ -2180,3 +2180,30 @@ int config_parse_ip_protocol(
         *proto = r;
         return 1; /* done. */
 }
+
+int config_parse_loadavg(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        int r;
+        loadavg_t *i = ASSERT_PTR(data);
+        assert(rvalue);
```

Comment:
Please insert an empty line between variable declarations and assertion. Also, not a strict rule, but we usually list variables longer line first. So,
```c
loadavg_t *i = ASSERT_PTR(data);
int r;

assert(rvalue);
```